content/03.prior-knowledge.representation-and-transfer.md

```diff
@@ -0,0 +1,55 @@
+(Here "unseen" refers to the fact that no AAV or medulloblastoma data were included in the training set.)
+PLIER is a matrix factorization approach that takes prior knowledge in the form of gene sets or pathways and gene expression data as input; PLIER utilizes regularization such that some latent variables learned by the model will align with input gene sets and, for those latent variables that are aligned with gene sets, latent variables will be associated with a limited number of gene sets [@doi:10.1038/s41592-019-0456-1].
+We demonstrated that training on larger collections of randomly selected samples produced models that captured a larger proportion of input gene sets and were more suitable for disentangling closely related signals (e.g., type I and type II interferon signaling), suggesting that larger training sets produced PLIER models that are more valuable for biological discovery [@doi:10.1016/j.cels.2019.04.003].
+
+Though models trained on general collections of transcriptomic data had more appealing properties, that alone does not guarantee suitability for describing rare diseases.
+To assess suitability, we must find ways to examine the relevance of learned features (latent variables) to the rare disease gene expression data.
+In Taroni et al., we found that the expression of latent variables that could be matched between the MultiPLIER model and a model trained on a rare disease dataset alone were well-correlated, particularly in the case of latent variables that were biologically relevant (i.e., significantly associated with input gene sets) [@doi:10.1016/j.cels.2019.04.003].
+Despite the absence of AAV from the training set, MultiPLIER was able to learn a latent variable where the genes with the highest contributions encode antigens that the antineutrophil cytoplasmic antibodies (ANCA) form against in AAV and with higher expression in a group of samples from patients with AAV that were reported to have more severe disease [@doi:10.1002/art.27398].
+The utility of this approach stems from the fact that biological processes are often _shared_ between conditions–the same ANCA antigen genes are components of normal neutrophilic granule development that is likely captured or assayed in the collection of transcriptomic data used for training.
+MultiPLIER has additional attributes that make it practical for studying rare diseases: not all latent variables learned by a PLIER model are associated with input gene sets, and therefore may capture technical noise separately from biological signal, and we can use a single model to describe datasets from multiple tissues or cohorts that are not obviously directly comparable instead of attempting to reconcile results from multiple models (see _05.heterogeneity.md_).
```

Comment:
Note to reviewers: Boy, there sure is a lot of information about MultiPLIER here. Although I do think there are multiple lessons to be learned from that paper, some of this may be better suited for #82.